resources/GraphQL.js

```diff
@@ -1,6 +1,24 @@
-const { readFileSync: read } = require('fs');
+const { readFileSync: read, readdirSync: readdir } = require('fs');
+const path = require('path');
+
+function loadGraphQL(target, type) {
+  let dir = `${__dirname}/graphql/${type}`
+  readdir(dir, {withFileTypes: true}).forEach((e) => {
+    if (e.isFile() && path.extname(e.name) == '.graphql') {
+      let name = path.basename(e.name, '.graphql');
+      target[name] = read(`${dir}/${e.name}`).toString();
+    }
+  });
+}
+
+let queries = {};
+let mutations = {};
+loadGraphQL(queries, "queries");
```

Comment:
Since `queries` and `mutations` are already global variable, I'd not pass them as parameters to avoid confusion.

Reply:
How do you imagine this implementation then? Currently `loadGraphQL` doesn't rely on any global variables. You can pass any object to get filled with GraphQL documents.

I don't see what confusion can happen here.
But actually we can just simply return them.

So
```js
let queries = getGraphQL("queries");
let mutations = getGraphQL("mutations");
```

Reply:
Yeah, that's better.